core/templates/account.html

```diff
@@ -0,0 +1,45 @@
+{% extends "newBase.html" %}
+{% load static %}
+{% block title %}My Profile{% endblock %}
+{%block head%}
+  <link rel="stylesheet" href="{% static 'core/profile.css' %}">
+{%endblock%}
+{% block content %}
+
+<main class="main-content">
+    <!-- Linked Accounts -->
+    <section class="section">
+        <h2 class="section-title">Linked Accounts</h2>
+        <div class="accounts-grid">
+            <div class="account-card">
+                <div class="card-header">
+                    <h3>Discord</h3>
+                    <p class="card-description">Connect your Discord account</p>
+                </div>
+                <div class="card-content">
+                    <button class="button button-primary">LINK ACCOUNT</button>
+                </div>
+            </div>
+            <div class="account-card">
+                <div class="card-header">
+                    <h3>GitHub</h3>
+                    <div class="user-info">
+                        <div class="avatar small">
+                            <img src="/placeholder.svg?height=20&width=20" alt="GitHub avatar">
+                        </div>
+                        <span>jasonappah</span>
```

Comment:
Hi jason

Reply:
sup

Reply:
Why is your name here lol

Reply:
oh this page shouldn't be linked to at all I think, mostly dependent on the account linking stuff